handlers/add_task.go

```diff
@@ -0,0 +1,40 @@
+package handlers
+
+import (
+	"time"
+
+	"github.com/go-openapi/runtime/middleware"
+
+	runtime "github.com/AliShan302/Task_Schedular"
+	"github.com/AliShan302/Task_Schedular/gen/restapi/operations"
+	"github.com/AliShan302/Task_Schedular/models"
+)
+
+// NewAddTask handles request for saving task
+func NewAddTask(rt *runtime.Runtime) operations.AddTaskHandler {
+	return &addTask{rt: rt}
+}
+
+type addTask struct {
+	rt *runtime.Runtime
+}
+
+// Handle the add task request
+func (d *addTask) Handle(params operations.AddTaskParams) middleware.Responder {
+	task := models.Task{
+		ID:          params.Task.ID,
+		Name:        params.Task.Name,
+		CreatedAt:   time.Time(params.Task.CreatedAt),
+		Status:      params.Task.Status,
+		Description: params.Task.Description,
+		Deadline:    time.Time(params.Task.Deadline),
+		Priority:    int(params.Task.Priority),
+		Assignee:    params.Task.Assignee,
+	}
+
+	if err := d.rt.Service().AddTask(&task); err != nil {
+		log().Errorf("failed to add task: %s", err)
+		return operations.NewAddTaskInternalServerError()
```

Comment:
you have defined 409 in swagger file addTask request, but I don't see it being returned anywhere in add task request